nmxact/tcp/tcp_sesn.go

```diff
@@ -0,0 +1,199 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package tcp
+
+import (
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/runtimeco/go-coap"
+
+	"mynewt.apache.org/newtmgr/nmxact/mgmt"
+	"mynewt.apache.org/newtmgr/nmxact/nmcoap"
+	"mynewt.apache.org/newtmgr/nmxact/nmp"
+	"mynewt.apache.org/newtmgr/nmxact/nmxutil"
+	"mynewt.apache.org/newtmgr/nmxact/omp"
+	"mynewt.apache.org/newtmgr/nmxact/sesn"
+)
+
+const MAX_PACKET_SIZE = 512 // Completely arbitrary.
+
+type TcpSesn struct {
+	cfg    sesn.SesnCfg
+	conn   net.Conn
+	txvr   *mgmt.Transceiver
+	isOpen bool
+}
+
+func NewTcpSesn(cfg sesn.SesnCfg) (*TcpSesn, error) {
+	s := &TcpSesn{
+		cfg: cfg,
+	}
+
+	var err error
+
+	s.txvr, err = mgmt.NewTransceiver(cfg.TxFilterCb, cfg.RxFilterCb, true,
```

Comment:
should it be mgmt.NewTransceiver(cfg.TxFilter, cfg.RxFilter, true,
		cfg.MgmtProto, 3)

?